qiskit/version.py

```diff
@@ -113,11 +113,34 @@ def _load_versions(self):
             self._version_dict['qiskit-ibmq-provider'] = ibmq.__version__
         except Exception:
             self._version_dict['qiskit-ibmq-provider'] = None
+        # TODO: Remove aqua after deprecation is complete and it is removed from
+        # the metapackage
         try:
             from qiskit import aqua
             self._version_dict['qiskit-aqua'] = aqua.__version__
         except Exception:
             self._version_dict['qiskit-aqua'] = None
+        try:
+            import qiskit_nature
+            self._version_dict['qiskit-nature'] = qiskit_nature.__version__
+        except Exception:
```

Comment:
Why not `ModuleNotFoundError` instead?

```suggestion
        except ModuleNotFoundError:
```

Or `ImportError` if `qiskit_nature.__version__` might stop existing (notice `ModuleNotFoundError` is `ImportError` subclass)

```suggestion
        except ImportError:
```

Reply:
This is correct as is. We never want this function to raise an exception since it's just gathering version info. If *any* exception is raised for any reason while importing an extra or while trying to access the version we want to just want to ignore that package.